doc/api/process.markdown

```diff
@@ -59,6 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
+This event is not guaranteed to fire in all situations. For example, if the
+process receives `SIGINT`, it may exit without firing this event.
```

Comment:
@Trott, please review line 247 below. It's the last bullet point under [the section on Signal Events](https://github.com/nodejs/node/blob/master/doc/api/process.markdown#signal-events)

I might say something like:
> The exit event will not fire if the process receives one of three program interrupts intended to override the runtime regardless of how deep call stack may be, i.e. `SIGINT`, `SIGTERM`, or `SIGKILL`.